collections/Laud_Misc/MS_Laud_Misc_306.xml

```diff
@@ -38,12 +37,38 @@
                   </altIdentifier>
                </msIdentifier>
                <physDesc>
-                  <p n="composite">Composite</p>
+                  <objectDesc form="codex">
+                     <supportDesc material="perg">
+                        <support>parchment</support>
+                        <extent>i (paper) + i (parchment) + i (medieval parchment) + 158 + ii (parchment) <dimensions type="leaf" unit="mm">
+                              <height>395</height>
+                              <width>305</width>
+                           </dimensions>
+                        </extent>
+                        <foliation>i–ii, 1–161, in pencil.</foliation>
+                        <collation>Quire signatures XV–XVIII and XXII in red span the two main parts (fols. 91r, 99r, 107r, 116r, and 148r), suggesting that the volume was mainly composed of quaternions, of which three are missing at the beginning. One would expect the 15th quaternion to start on the 113th leaf, not the 90th, so about 24 leaves are apparently missing. <secFol>Et verum</secFol></collation>
+                     </supportDesc>
+                  </objectDesc>
+                  <bindingDesc>
+                     <binding notAfter="1645" notBefore="1636">
+                        <p>Laudian binding. Sewn on six bands, laced into pasteboards covered with polished brown leather, each cover stamped in gilt with the arms of Archbishop Laud, with vestiges of two ties; the spine with most of a paper label inscribed with a title in the 17th-century; the joints and spine repaired.</p>
+                     </binding>
+                  </bindingDesc>
                </physDesc>
+               <history>
+                  <origin>
+                     <origDate calendar="Gregorian" notAfter="1200" notBefore="1100">12th century</origDate>
+                     <origPlace>
+                        <country key="place_7000084">German</country>
+                     </origPlace>
+                  </origin>
```

Comment:
Not sure about Peter putting this date here; seems better to date the component units individually